Missionframework/modules/15_build/fnc/fn_build_setupPlayerActions.sqf

```diff
@@ -20,7 +20,7 @@
 // Actions avalible LOCALLY to player
 if(hasInterface) then {
     // Build action
-    private _buildCondition = '_target == _originalTarget && !(_originalTarget getVariable ["KPLIB_fob", ""] in ["", "KPLIB_eden_startbase_marker"])';
+    private _buildCondition = '_target == _originalTarget && !(_originalTarget getVariable ["KPLIB_fob", ""] in ["", "KPLIB_eden_startbase_marker"]) && [player, "Build"] call KPLIB_fnc_permission_getPermission';
```

Comment:
```suggestion
    private _buildCondition = '
        (_target == _originalTarget) &&
        !(_originalTarget getVariable ["KPLIB_fob", ""] in ["", "KPLIB_eden_startbase_marker"]) &&
        ([player, "Build"] call KPLIB_fnc_permission_getPermission)
    ';
```
It's already fine, but after the first suggestion I thought these "longer &&/|| conditions" could be formatted like that in the future or replace as soon as we see some.